.github/workflows/coverity_scan.yml

```diff
@@ -0,0 +1,60 @@
+name: coverity-scan
+on:
+  workflow_dispatch: # because sometimes you just want to force a branch to have tests run
+  push:
+    branches:
+      - "**"
+    paths:
+      - "backend/**"
+      - .github/workflows/coverity_scan.yml
+  schedule:
+    - cron: '0 18 * * *' # Daily at 18:00 UTC
+
+defaults:
+  run:
+    working-directory: ./backend
+
+jobs:
+  latest:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "14"
+      - name: Install dependencies
+        run: pushd ../frontend/ && yarn install && popd
```

Comment:
Can use cache.

Reply:
Not sure what this means. More context?

Reply:
You can cache node dependencies as well as Java ones (and you can leverage the cache we already use)

Yoinked from smoketestBackend.yml:

``` - name: Cache npm local files
        uses: actions/cache@v2
        with:
          path: |
            ./frontend/node_modules
          key: npm-${{env.NODE_VERSION}}-${{ hashFiles('frontend/yarn.lock', 'frontend/package.json') }}
```